docs/testcases/UI/Run_tools_in_sandbox/3052_report_run_tags_timepoint/3052_1.md

```diff
@@ -0,0 +1,30 @@
+# Report timestamp when a PRESSURE tag was assigned to a run
+
+Test verifies that report timestamp when PRESSURE tag was assigned to a run.
+
+**Prerequisites**:
+- admin user
+
+**Preparations**:
+1. Login as the admin user from the prerequisites
+2. Set the System Preference `system.memory.consume.threshold` value as `20`
+3. Set the System Preference `system.resource.monitoring.period` value as `60000`
+4. Check that `system.run.tag.date.suffix` preference has value `_date`
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open the **Tools** page | |
+| 2 | Select test tool | | 
+| 3 | Launch a selected tool with default settings | |
+| 4 | Open the **Runs** page | |
+| 5 | Wait about 2 minutes | ***PRESSURE*** label is shown near the just-launched run name |
```

Comment:
Waiting time can vary. I suggest instead the step like "wait until the SSH hyperlink will appear"